hudi-common/src/main/java/org/apache/hudi/common/config/TypedProperties.java

```diff
@@ -37,10 +38,12 @@ public TypedProperties(Properties defaults) {
     super(defaults);
   }
 
-  private void checkKey(String property) {
-    if (!containsKey(property)) {
+  private boolean checkKey(String property) {
```

Comment:
`checkKey` would be rename to `containsKey`

Reply:
I think the checkkey is better。That was its original name。containsKey in  parent class，

Reply:
> I think the checkkey is better。That was its original name。containsKey in parent class，

`checkKey` should not return boolean and should throw exception from semantically. ` containsKey` or `keyExists` should return boolean.

Reply:
ok,i see,i get it